wai-extra/Network/Wai/Test.hs

```diff
@@ -5,6 +5,7 @@ module Network.Wai.Test
     ( -- * Session
       Session
     , runSession
+    , runSessionWith, ClientState(..), initState
```

Comment:
Please add `@since` comments with a new minor version bump for all newly exposed identifiers, as well as a modified .cabal file and updated ChangeLog.